packages/obj-logger/src/utils/pretty-print.ts

```diff
@@ -0,0 +1,17 @@
+import { COLORS } from "./colors";
+import { LogEntry, LogLevel } from "@scramjet/types";
+
+const COLOR_MAP: { [ key: string]: string } = {
+    TRACE: COLORS.Dim,
+    DEBUG: COLORS.FgBlue,
+    INFO: COLORS.FgCyan,
+    WARN: COLORS.FgYellow,
+    ERROR: COLORS.FgRed,
+    FATAL: COLORS.FgMagenta
+};
+
+const getLevelColor = (level: LogLevel = "INFO") => `${COLOR_MAP[level]}${level?.padEnd(5)}${COLORS.Reset}`;
+
+export const prettyPrint = (obj: LogEntry) => {
```

Comment:
We should have another version with no color if color was not turned on, or logs are sent to non tty.

Reply:
https://github.com/chalk/supports-color

Some inspiration...